tests/test_plugin.py

```diff
@@ -261,3 +263,186 @@ def test_crud_for_navigation_plugin_when_versioning_disabled(self):
 
         # Now delete the plugin from the page and assert
         self._delete_nav_plugin_and_assert(new_placeholder, new_plugin)
+
+    def test_menu_cache_invalidate_after_menuitem_creation(self):
```

Comment:
I think this is actually a test for whether the cache is invalidated after edit rather than after creation, right?